### ISSUE TITLE
Stricter traderoute creation

### ISSUE THREAD

**Issue by IRampagingRhin0** [andrewfullard/PyGCEditor#59]
Currently if any new traderoute is made it'll output them

Now if you make a new one and then decide you don't like it or redo the area, then that traderoute has been created, but is now unused

Maybe a check to see if new routes are actually used and delete any new ones when saving

**Comment by ThrawnsRevenge** [comment]
I don't think this should be an automatic thing, and this would be somewhat detrimental for certain uses of the editor (ie making multiple sizes of the same map where sometimes when changing certain elements, this would mean I end up re-creating the same traderoute multiple times if I have a connection in one map, remove a planet, and then add it in the next. My hope as far as trade routes go is to make the editor import and export the whole file (so rather than saving just new routes, it saves all of them). We can make a separate save option just for traderouts to delete any unused ones.
